packages-web/maps/typings/shared.d.ts

```diff
@@ -0,0 +1,40 @@
+import { HeightUnitEnum, WidthUnitEnum } from "./MapsProps";
```

Comment:
You don't have to use `.d.ts` extension to define types. It can be a normal `.ts`

Reply:
Typing generator is using .d.ts as default output.. I was just keeping consistency

Reply:
fair